### PR TITLE
Pphoje/bugfix

### PR DESCRIPTION
**Pull request by filipecosta01**

### CODE REVIEW

**Comment by daniloaf** [comment]
Which bug is this related to? Could you create a new issue with the label `bug`, please? Thanks!

File: README.rdoc [hunk 1]
```diff
@@ -1,28 +1,44 @@
 == README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Are you at Campina Grande/PB during São João and are looking for the events of Parque do Povo?
```
Comment (felipevieira) [inline README.rdoc]: ... and also looking?

File: README.rdoc [hunk 2]
```diff
@@ -1,28 +1,44 @@
...
 
-Things you may want to cover:
+We can help you to find the shows of Parque do Povo - Campina Grande/PB with this Telegram Bot, pphoje_bot.
 
-* Ruby version
+If you are a Telegram user, you can include this bot into a channel or write commands directly on a private chat.
 
-* System dependencies
+We support the commands:
```
Comment (felipevieira) [inline README.rdoc]: the following commands

File: README.rdoc [hunk 3]
```diff
@@ -1,28 +1,44 @@
...
-* Configuration
+* /start - To start the bot
 
-* Database creation
+* /help  - To check the supported commands
 
-* Database initialization
+* /shows {day_number} {week_day} {next_day} - To check the shows for a specific date. The arguments are optional and you can inform one of the commands as the example.
+{day_number} -> Integer value, representing the day of a month. [1, 2, 3, 4, ..., 30]
+{week_day}   -> String value, representing the day of a week. [segunda, terca|terça, quarta, quinta, sexta, sabado|sábado, domingo]
```
Comment (felipevieira) [inline README.rdoc]: expandir pros dias da semana em inglês?

File: README.rdoc [hunk 4]
```diff
@@ -1,28 +1,44 @@
...
+{next_day}   -> String value, representing the day after today. [amanha|amanhã]
```
Comment (felipevieira) [inline README.rdoc]: expandir pra tomorrow?

File: README.rdoc [hunk 5]
```diff
@@ -1,28 +1,44 @@
...
 
-* How to run the test suite
+Example of usage:
+/shows 21 -> return the shows for the 21th day of the current month
+/shows amanha  -> return the shows for the next day
+/shows segunda -> return the shows on Monday (current day, if it's Monday or next Monday otherwise)
```
Comment (felipevieira) [inline README.rdoc]: returns

File: README.rdoc (hunk 1 above)
Comment (felipevieira) [inline README.rdoc]: Do you want to stay tuned about the big events that will be happening at Parque do Povo during the Greatest São João of the World - Campina Grande?

File: README.rdoc [hunk 6]
```diff
@@ -1,28 +1,44 @@
 == README
```
Comment (daniloaf) [inline README.rdoc]: I think it's missing the sessions `License` and `Contributing`

File: app/controllers/telegram_webhooks_controller.rb [hunk 7]
```diff
@@ -39,24 +24,32 @@ def shows(*args)
...
+
         @shows.each do |show|
-          response_message += "-> " + show.nome + "\n"
-          response_message += "- Show confirmado: " + (show.confirmado ? "Sim" : "Não") + "\n"
-          response_message += "- Informações da banda: " + show.link_artista + "\n\n"
+          response_message += t(:event_title, :eventTitle => show.name)
+
+          eventConfirmed = show.is_confirmed ? t(:confirmed) : t(:not_confirmed)
+          response_message +=  t(:event_confirmed, :eventConfirmed => eventConfirmed)
+          response_message +=  t(:event_band_info, :moreInfo => show.link_band)
```
Comment (felipevieira) [inline app/controllers/telegram_webhooks_controller.rb]: try to unify the i18n labels nomenclature. +1 for underscore case (more_info) in this case

File: config/locales/en.yml [hunk 8]
```diff
@@ -1,23 +1,25 @@
...
+  week_day_friday     : "fri"
+  week_day_saturday   : "sat"
+  week_day_sunday     : "sun"
+  week_tomorrow       : "tomorrow"
+
+  custom_data_format   : "%{month}/%{day}"
+
+  check_usage_command  : "Verify the usage instructions using the command /help"
+
+  error_invalid_days  : "The Greatest São João of the world take place between June 3rd and July 3rd. Inform a valid day to get more info."
```
Comment (felipevieira) [inline config/locales/en.yml]: shouldn't we use "greatest" (without capitalization)?

**Pull request merged**